_includes/pbc/all/install-features/202410.0/install-the-customer-account-management-feature.md

```diff
@@ -131,6 +131,27 @@ class CustomerConfig extends SprykerCustomerConfig
 }
 ```
 
+4. Optional: To speed-up the query response time for the customer registration, adjust the `CustomerConfig::isCustomerEmailValidationCaseSensitive()` to return `true`:
```

Comment:
please describe drawbacks of this action if we have any

Reply:
there are no drawbacks, just query response time optimisation, which was already described